frontend/src/views/home/HomeNotLoggedinView.vue

```diff
@@ -0,0 +1,52 @@
+<template>
+  <div class="row">
+    <logo-image class="child1"></logo-image>
+    <div class="child2">
+      <home-not-loggedin-component></home-not-loggedin-component>
```

Comment:
Exactly that's what I meant, We should make a condition if we  have the user token, just like that 
```suggestion
      <template v-if="user.token && user.loggedIn">
          <home-component></home-component>
      <template/>
      <template v-else>
          <home-not-loggedin-component></home-not-loggedin-component>
      <template/>
```

for now, you can start with setting a dummy data in local storage till integrating the auth API and storing the token in the cookie